zerver/webhooks/pingdom/doc.md

```diff
@@ -1,24 +1,33 @@
+# Zulip Pingdom integration
+
 Zulip supports integration with Pingdom and can notify you of
 uptime status changes from your Pingdom dashboard.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. Next, go to the following URL:
+1. In the main menu, go to **Integrations** under the **Integrations**
+    menu, and click on **Add integration**.
 
-    `https://my.pingdom.com/reports/integration/settings`
-
-1. Create your integration by clicking on the **Add Integration** button
-    and filling the form like so:
+1. Configure the integration as follows and click **Save integration**:
```

Comment:
I think it might be better to verbalize these settings, and remove the image.
For example: people who use screen readers might find it helpful this way.